zulipterminal/ui.py

```diff
@@ -164,6 +164,9 @@ def keypress(self, size: Tuple[int, int], key: str) -> Optional[str]:
         elif is_command_key('ALL_STARRED', key):
             self.model.controller.show_all_starred(self)
             self.body.focus_col = 1
+        elif is_command_key('ALL_MENTIONS', key):
+            self.model.controller.show_all_mentions(self)
+            self.body.focus_col = 1
```

Comment:
Perhaps a little minor, but perhaps squash this and the README/keys commit, to avoid a potential state where the docs/help don't match the UI (even if the visual button doesn't exist, the key does work!)

Reply:
Good catch!